server/index.js

```diff
@@ -37,9 +41,65 @@ app.use(parser.json());
 // serve client files
 app.use('/', express.static(__dirname + './../client/dist'));
 app.use('/shop/:productId/:styleId', express.static(__dirname + './../client/dist'));
-app.use('/', router);
+
+// Moved CRUD here because data was getting mangled
+app.get('/product/:id', async(req, res) => {
+  try {
+    const { id } = req.params;
+    const data = await pool.query('SELECT * FROM products WHERE productId = $1', [id]);
+
+    if (!data) {
+      return res.status(400).send('No such product');
```

Comment:
as simple as it is, this error msg will probably make Leslie sleep happier at night

Reply:
And Nu Ralik knows Leslie could use more sleep!